src/powerquery-formatter/formatV2.ts

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as PQP from "@microsoft/powerquery-parser";
```

Comment:
Nit: cleanup imports so that it first places external dependencies, blank space, and end with local imports.